github-actions/action.yml

```
@@ -18,36 +18,35 @@ inputs:
   exclude:
     description: |-
       A list of paths (one per line) to exclude from the generation. Glob
-      patterns are accepted.
-      For reference, Ruby's Dir.glob is used:
-        https://ruby-doc.org/core-2.6.3/Dir.html#method-c-glob
+      patterns are accepted. For reference, Ruby's Dir.glob is used:
+      https://ruby-doc.org/core-2.6.3/Dir.html#method-c-glob.
     required: false
     default: |-
       third-party/**/*
 outputs: {}
 runs:
   using: composite
   steps:
-  - run: |-
-      if [ -z '${{ github.head_ref }}' ]; then
-        >&2 echo "Not a pull request..."
-        exit 1
-      fi
-    shell: bash
   - run: ${{ github.action_path }}/generate.rb
     shell: bash
     env:
       EXCLUDE: ${{ inputs.exclude }}
   - run: |-
       set -o errexit -o nounset -o pipefail
-
       mapfile -d '' -t < <(git ls-files -z --deleted --modified --others **/*README.md)
-      if [ "${#MAPFILE[@]}" -gt 0 ]; then
+      echo "::set-output name=files::${MAPFILE[@]}"
+      echo "::set-output name=count::${#MAPFILE[@]}"
```

```suggestion
      echo "::set-output name=files::${MAPFILE[*]}"
      echo "::set-output name=count::${#MAPFILE[*]}"
```

Because you're using them as one string (as far as I can tell), it shouldn't be using `@`.  Unless it's actually doing something _really_ confusing… (Technically the second one doesn't matter.)

---

Copy-pasta from my previous code.